Jenkinsfile

```diff
@@ -21,7 +21,7 @@ String contractsEnvFile       = 'env.template'
 String solcVersion            = '0.8.14'
 String nodeVersion            = '14.16.1'
 String gitHubUser             = 'sorabot'
-String gitHubRepo             = 'github.com/soramitsu/sora2-substrate.git'
+String gitHubRepo             = 'github.com/soramitsu/sora2-network.git'
```

Comment:
```suggestion
String gitHubRepo             = 'github.com/sora-xor/sora2-network.git'
```

Reply:
Yes, but it is just merge PR.
I think it is better to create another PR to develop branch with your changes

Reply:
https://github.com/sora-xor/sora2-network/pull/304